src/components/PasswordReset/index.jsx

```diff
@@ -0,0 +1,114 @@
+import { Alert, Input } from "antd";
+import React, { useState } from "react";
+import { validateEmail } from "../../functions";
+import "./index.css";
+
+const Login = (props) => {
```

Comment:
this is PasswordReset/index but this component is called Login

Reply:
it will be updated in the new PR.